doc/basics/testbase_tutorial_1.py

```diff
@@ -26,16 +26,18 @@ def on_timeout(self) -> None:
         self.overlay.timed_out = True
 
 
+class MyCommunitySettings(CommunitySettings):
```

Comment:
Shouldn't this be added to `MyCommunity.settings_class`?

Reply:
Hold on, this entire test class is not passing 😨

Reply:
Strange. I wonder why the validation tests did not catch that.. 🤔